app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError(
+            "Invalid command format. "
+            'Expected format: "cp <source_file> <destination_file>"')
+    source_file = parts[1]
+    destination_file = parts[2]
+
+    if source_file != destination_file:
+        try:
+            with (open(source_file, "r") as file_in,
+                  open(destination_file, "w") as file_out):
+                file_out.write(file_in.read())
+        except FileNotFoundError:
```

Comment:
FilenotFoundError is a sublcass for OSError, IOError is an alias for OSError - so you can simply use OSError here and it will cover all errors that might happen during file handling